Assets/PresetsPerFolder/PresetPostProcessor.cs

```diff
@@ -0,0 +1,219 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEditor.Experimental;
+using UnityEditor.Presets;
+using UnityEngine;
+
+namespace PresetsPerFolder
+{
+    /// <summary>
+    /// This class is applying Presets assets automatically to each assets added to the folder containing the Preset or any subfolder.
```

Comment:
Suggestion: "This sample class applies presets automatically to assets in the folder containing the Preset, & any subfolders."